services/indexes/pvm/writer.go

```diff
@@ -225,7 +238,7 @@ func (w *Writer) ParseJSON(txBytes []byte) ([]byte, error) {
 			BlockType: &btypeS,
 		}, nil
 	}
-	proposerBlock, err := block.Parse(txBytes)
+	proposerBlock, _, err := block.Parse(txBytes) // double check if the blueberry bool is relevant
```

Comment:
`block.Parse(txBytes)` wut

Reply:
No idea what is going on here either to be clear